### ISSUE TITLE
I keep getting the same 2 issus first is not so important

### ISSUE THREAD

**Issue by FoxFireZander09** [Jerakin/Pokedex5E#740]
First issue is when ever I save a move the Casting time turns from 1 Action into Instantaneous and the middle move power turns to none from what ever I put it to 

Second issue is when ever I click update existing on the Meta Data tab it comes up with the below in a box and crashes my app

Version: 0.2.8
OS: Windows-10-10.0.19041-SP0 
Type: <class 'TypeError'>
Value: 'SimpleList' object is not iterable
Traceback:
File "creator\child_views\metadata_tab.py", line 65, in update_package

File "creator\child_views\list_view.py", line 14, in __init__

**Comment by magroader** [comment]
I assume this is for the Fakemon creator?

**Comment by FoxFireZander09** [comment]
Yes these r the problems I am having for the ones I am trying to create
